tock/projects/models.py

```diff
@@ -185,6 +185,12 @@ def active(self):
     def inactive(self):
         return self.get_queryset().filter(active=False)
 
+    def excluded_from_billability(self):
+        return self.get_queryset().filter(exclude_from_billability=True)
+
+    def non_billable(self):
+        return self.get_queryset().filter(exclude_from_billability=False, accounting_code__billable=False)
```

Comment:
I wonder -- not advocating, just wondering -- if we should add some managers for these things.

Reply:
Can you elaborate more here?  Probably will work on it separately from this PR.

Reply:
Just wondering if it would be useful to have `Projects.non_billable.all()` and such. Not looking for any change, just thinking about it.